src/features/Send/useCases/ListAmountsToSend/AddToken/SelectTokenFromListScreen.tsx

```diff
@@ -256,6 +260,65 @@ const EmptySearchResult = () => {
   )
 }
 
+const Counter = ({fungibilityFilter}: {fungibilityFilter: FungibilityFilter}) => {
+  const {search: assetSearchTerm, visible: isSearching} = useSearch()
+  const strings = useStrings()
+  const wallet = useSelectedWallet()
+  const filteredTokenInfos = useFilteredTokenInfos({fungibilityFilter})
+  const {nfts} = useNfts(wallet)
+
+  if (!isSearching && fungibilityFilter === 'all') {
+    const total = filteredTokenInfos.length
```

Comment:
just to keep consistent with component name
```suggestion
    const counter = filteredTokenInfos.length
```

Reply:
but if it receives {counter} as prop, that it can be dismissed